src/oas.md

```diff
@@ -3274,46 +3274,85 @@ For examples using `attribute` or `wrapped`, please see version 3.1 of the OpenA
 
 ###### No XML Object
 
-Basic string property (`nodeType` is `element` by default):
+Basic string property without an XML Object, using `serializedValue` (the remaining examples will use `externalValue` so that the XML form can be shown with syntax highlighting):
```

Comment:
am I right in thinking the serializedValue PR would be a dependency of this PR? (no drama if it is, we just need to make sure to review the other one first)